index.rb

```diff
@@ -2,16 +2,17 @@
 
 # library = Library.new
```

Comment:
Do not use comments. Here you can move this to service and then use it with some `if` condition (you can use ENV variable)